tcms/testplans/static/testplans/js/get.js

```diff
@@ -24,16 +24,22 @@ $(document).ready(function() {
     const perm_remove_tag = $('#test_plan_pk').data('perm-remove-tag') === 'True';
     tagsCard('TestPlan', testPlanId, {plan: testPlanId}, perm_remove_tag);
 
-    jsonRPC('TestCase.filter', [{'plan': testPlanId}], function(data) {
-        for (var i = 0; i < data.length; i++) {
-            var testCase = data[i];
-            allTestCases[testCase.id] = testCase;
-        }
-        drawTestCases(Object.values(allTestCases), testPlanId, permissions);
+    jsonRPC('TestCase.sortkeys', {'plan': testPlanId}, function(sortkeys) {
+        jsonRPC('TestCase.filter', {'plan': testPlanId}, function(data) {
+            for (var i = 0; i < data.length; i++) {
+                var testCase = data[i];
+
+                testCase.sortkey = sortkeys[testCase.id];
+                allTestCases[testCase.id] = testCase;
+            }
+            // TODO: UI shows `Summary` as selected for initial sort but that isn't true,
+            // not sure if I want to change it now.
```

Comment:
I think thats ok, because we have a button to trigger sorting.

Reply:
The issue that we have here is that rows are now sorted by sortkey but the UI still shows Summary as selected by default. When the a-z/z-a button is clicked it will change the sort direction and reorder by Summary at the same time. This may be confusing to people at first.